bazel/linux/templates/runner.template.sh

```diff
@@ -161,6 +161,13 @@ test "$estatus" == 0 || {
     echo 1>&2 "===== emulator exited with non zero status - check logs above ===="
     exit "$estatus"
 }
+
+if [ ${XML_OUTPUT_FILE+x} ]; then
+  # The program in the emulator has created a junit.xml. Move it to the
+  # expected location.
+  mv "$OUTPUT_DIR/junit.xml" $XML_OUTPUT_FILE
```

Comment:
2 things here:
1) `"$XML_OUTPUT_FILE"` (quotes around the $ variable)
2) logic seems fishy? Should the if instead test to see if "$OUTPUT_DIR/junit.xml" exists, and if it does, copy it in the right place?